vendor/assets/javascripts/mediaelement/mediaelement-and-player.js

```diff
@@ -3770,7 +3770,7 @@ var MediaElementPlayer = function () {
 
 			doAnimation = doAnimation === undefined || doAnimation;
 
-			if (forceHide !== true && (!t.controlsAreVisible || t.options.alwaysShowControls || t.paused && t.readyState === 4 && (!t.options.hideVideoControlsOnLoad && t.currentTime <= 0 || !t.options.hideVideoControlsOnPause && t.currentTime > 0) || t.isVideo && !t.options.hideVideoControlsOnLoad && !t.readyState || t.ended)) {
+			if (forceHide !== true && (!t.controlsAreVisible || t.options.alwaysShowControls || t.paused && t.readyState === 4 && (!t.options.hideVideoControlsOnLoad && t.currentTime <= 0 || !t.options.hideVideoControlsOnPause && t.currentTime > 0) || t.isVideo && !t.options.hideVideoControlsOnLoad && !t.readyState || t.ended || t.controls.matches(':hover'))) {
```

Comment:
If this eventually becomes a PR to mediaelement, they might want an alternative selector to `matches()`... worth a shot, but looks like IE and mobile (android at least) might handle differently?  Not really sure, just a thought.  https://developer.mozilla.org/en-US/docs/Web/API/Element/matches